.gitignore

```diff
@@ -66,3 +66,6 @@ typings/
 
 # snapshots
 __snapshots__
+
+# package-lock
+package-lock.json
```

Comment:
don't ignore this file, it is same as yarn.lock file

Reply:
If we gonna use it to lock the dependency versions, maybe we can commit it to github.